### ISSUE TITLE
Automatic Photo Upload

### ISSUE THREAD

**Issue by MuscleRumble** [cryptomator/cryptomator-ios#20]
"Automatic Photo Upload" has been requested by some users already and I think it's quite self-explanatory. Should be similar to the Dropbox app.

**Comment by masvil** [comment]
Will you implement same options as Dropbox? I mean:
Upload Videos on/off
Use cellular data on/off
Background Uploading on/off

**Comment by MuscleRumble** [comment]
I haven't looked into the details yet, but I would've looked into Dropbox as a reference anyway. These options totally make sense! So yes, I'll try to achieve the same functionality. I'm still not sure when this feature will make it into Cryptomator, looks like it could take some time to implement it.

**Comment by masvil** [comment]
I agree: DropBox is the only iOS app to look at. Many cloud apps has this feature, but it's reliable only in DropBox. If you will release it then our privacy will be VERY increased!

**Comment by overheadhunter** [comment]
I wouldn't add the "Upload Videos" switch. As the user selects files explicitly, he will most likely know the implications of uploading a video.

**Comment by MuscleRumble** [comment]
No, this isn't the point of this issue. No files are explicitly selected by the user, photos (and optionally videos) will be uploaded automatically.

**Comment by FFornix** [comment]
Any update regarding the topic?

**Comment by tobihagemann** [comment]
Sorry, there's no update here. We'll plan version 1.3 soon, but I don't think this feature will make it on the list. You can definitely show your interest in this feature by giving it a thumbs up in my original post. There are currently other feature requests that definitely have more demand.

**Comment by LeadManPL** [comment]
I know development takes time, but 5 months passed. Any chance this will be released soon? :)

**Comment by 50kudos** [comment]
Question: Does this mean a photo or video taken will be encrypted on-the-fly and put it to the vault? If no, is it possible to prompt cryptomator after a photo has taken, so I type a password then it gets saved, or use wake-lock somehow?

**Comment by psmarcin** [comment]
Any updates?

**Comment by Trufax** [comment]
Any plans in adding this feature?

**Comment by awehrfritz** [comment]
Automatic photo upload would be my primary use-case for this app and pretty much the only reason for me to hold-off on purchasing the app.

Any chance this will be implemented soon? 

I assume this would need to be implemented in the parts that are not open source, or is that code somewhere available so I could look into the implementation of this feature?

**Comment by jessse4216** [comment]
Same here

**Comment by integer256** [comment]
Any updates on this feature?

**Comment by yakuraku** [comment]
I'd really love to see this feature, any update on this?
